docs/src/main/paradox/user/lang/kotlin/gettingstarted.md

```diff
@@ -36,197 +36,35 @@ libraryDependencies += "io.cloudstate" % "cloudstate-kotlin-support" % "$cloudst
-                        <configuration>
-                            <sourceDirs>
-                                <sourceDir>${project.basedir}/src/test/kotlin</sourceDir>
-                                <sourceDir>${project.basedir}/src/test/java</sourceDir>
-                            </sourceDirs>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
+## Simple example
 
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.5.1</version>
-                <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
-                </configuration>
-                <executions>
-                    <!-- Replacing default-compile as it is treated specially by maven -->
-                    <execution>
-                        <id>default-compile</id>
-                        <phase>none</phase>
-                    </execution>
-                    <!-- Replacing default-testCompile as it is treated specially by maven -->
-                    <execution>
-                        <id>default-testCompile</id>
-                        <phase>none</phase>
-                    </execution>
-                    <execution>
-                        <id>java-compile</id>
-                        <phase>compile</phase>
-                        <goals> <goal>compile</goal> </goals>
-                    </execution>
-                    <execution>
-                        <id>java-test-compile</id>
-                        <phase>test-compile</phase>
-                        <goals> <goal>testCompile</goal> </goals>
-                    </execution>
-                </executions>
-            </plugin>
+A minimal Kotlin example using the [Google Jib Docker Maven Plugin](https://github.com/GoogleContainerTools/jib/) 
+to generate the [Docker](https://www.docker.com/) image for a shopping cart service, is shown below:
 
-            <plugin>
-                <groupId>org.xolstice.maven.plugins</groupId>
-                <artifactId>protobuf-maven-plugin</artifactId>
-                <version>0.6.1</version>
-                <configuration>
-                    <protocExecutable>/usr/local/bin/protoc</protocExecutable>
-                    <protocArtifact>com.google.protobuf:protoc:3.9.1:exe:${os.detected.classifier}</protocArtifact>
-                </configuration>
-                <executions>
-                    <execution>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>compile</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
+Maven
+: @@@vars
+@@snip [Main.kt](/docs/src/test/kotlin/docs/user/gettingstarted/pom.xml) { #pom-xml }
+@@@
```

Comment:
`@@@vars` is only needed when using paradox's variable substitution. Shouldn't be needed here.

```suggestion
: @@snip [Main.kt](/docs/src/test/kotlin/docs/user/gettingstarted/pom.xml) { #pom-xml }
```

Reply:
As the xml excerpt was removed, the library version remained in the file. How will the replacement occur in this case?

Reply:
Okay, I didn't see any paradox variables in the pom.xml, and I don't think that paradox applies variable substitution to included snippets anyway.

What we've done before (in other projects using paradox) is run a variable substitution on build files first, which then get tested, and the generated files get pulled into the docs via snippets. We should add support for something like this. Will become obvious once we start testing the snippets anyway.

Reply:
Captured this in an issue (#306) so we can follow up.

Reply:
Given that the other docs (like java and kotlin support) currently embed the build description samples so they can have var substitutions, I think it makes sense to just do that here for now too. So revert back to how this PR was originally. I can follow up with supporting a substitution step for files later.